### ISSUE TITLE
LMMS and sight-impaired users 

### ISSUE THREAD

**Issue by musikBear** [LMMS/lmms#1935]
First i would ask you to read this: 
http://lmms.io/forum/posting.php?mode=quote&f=15&p=8643
Especially this
> So it's quite weird - sound is a natural means of self-expression of blind people, but sound manipulation software is heavily mouse oriented, thus inaccessible for such people. Very sad :(

really made me think. 'How right he is'
What are your opinion on this?
Could lmms be the first and only daw, that actually focused this, and gave sight-impaired users, a chance to make music on computer?
For my self, i have to say, that a lot of font-sizes and color-schemes, are just about to be issues, and i also often have to bend forward, in order to read a bar-posistion, or a readout from a dial, but those kind of issues could be addressed with more css-options, and are completely insignificant, compared to what OP tell us about.

**Comment by Sti2nd** [comment]
I doubt LMMS will evolve into a DAW for blind people, but allowing to jump around with some keys (and then blind persons can hear the description text being read by the OS) is within reach.

**Comment by musikBear** [comment]
I think that anything that could facilitate music-making  to sight-impaired, would be highly interesting, but there are so much else that is wanted in lmms, i know that..

**Comment by tresf** [comment]
@musikBear please post correct links when linking to the forums.

**Comment by tresf** [comment]
This whole bug report should be cleaned up.  The OPs question needs not be paraphrased in this case.  Please take better care when submitting bug reports.  We're already inundated with 10 new bugs each day, the least you could do is put more than 30 seconds into writing it (or direct the users here, as the OP seems to better appreciate proper formatting technique).

**Comment by tresf** [comment]
After reading the original request, it seems to have a lot of similarities with #1421 (albeit many other accessibility improvements to meet the OPs needs).

**Comment by musikBear** [comment]
> P.S. @musikBear please post correct links when linking to the forums. In addition, this whole bug report should be cleaned up. The OPs question needs not be paraphrased in this case. Please take better care when submitting bug reports. We're already inundated with 5-10 new bugs each day, the least you could do is put more than 30 seconds into writing it (or direct the users here, as the OP seems to better appreciate proper formatting technique).

C'ON -the link was correct - @Sti2nd  followed it - now it's bugged, but that has to be the forum 
Does it *really* look like i spend <30 sec ?!

**Comment by tresf** [comment]
> the link was correct - @Sti2nd followed it 

No, it is clearly wrong.  I left the old one around so you can try it.  Paste it into a private/incognito window.  It doesn't work.  @Sti2nd is a forum admin, so it worked for him (as did for me).

**Comment by tresf** [comment]
> Does it really look like i spend <30 sec ?!

I'm sorry if this came off as crude, but the formatting you use for your bug reports often makes them hard to read.  In this case, TomVal's description of his problem was much easier to read then yours.

His description:

> Is there any effort to make LMMS accessible for people with low vision, who still can use their sight, but not as much as sighted people?

Your description:

> Could lmms be the first and only daw, that actually focused this, and gave sight-impaired users, a chance to make music on computer?

:+1:

**Comment by unfa** [comment]
I guess we'd need a small team of dedicated developers who would work just on this. As we're generally short on coding people, this is probably not happening very soon :(

**Comment by musikBear** [comment]
This is one lmms could do something better rather fast
![paste](https://cloud.githubusercontent.com/assets/6662066/7812453/902abbd8-03b2-11e5-97fa-c5dbbb5b94aa.png)
When loudness is dimmed, it is as if the notes blend in with the scale-display, and the notes are really difficult to see -Even for peeps with normal sight 
- a simple style-sheet option could fix this?

**Comment by tresf** [comment]
> This is one lmms could do something better rather fast

Knock yourself out. :)

**Comment by Spekular** [comment]
I think a voice describing the current position could make piano roll very accessible for the vision impaired. It shouldn't require too many voice lines, and could be very configurable.

> State the current note:
> 
> - When my selected note or position (time) changes
> - When my selected note changes
> - When I press the Speak key
> 
> State the current position
> 
> - When my selected note or position changes
> - When my selected position changes
> - When I press the Speak key
> 
> State the current bar
> 
> - When my selected note or position changes
> - When my selected position changes
> - When moving from one bar to another
> - When I press the Speak key
> 
> [] Play a beep when a change occurs without speech
> 
> [] Simplify position (Two sixteenths -> One eight)

**Comment by OzancanKaratas** [comment]
Please update Qt framework to latest version. Qt 5 is containing accessibility objects, like IAccessible 2.

**Comment by tresf** [comment]
We have https://github.com/LMMS/lmms/issues/2611.

**Comment by OzancanKaratas** [comment]
I sent a message to lmmsservice on Transifex. Please add me as coordinator into Turkish.

**Comment by musikBear** [comment]
Today i learned that win10 at least has a screen-doubler-feature.
It is activated with 
WinKey then tab '+' sign
This launch a generalized Accessibility-gadget:
![image](https://user-images.githubusercontent.com/6662066/157958394-a725b51e-9c63-4983-8fb8-ebcaa970cffd.png)
It is the first part
- 100% +
That can help a lot!
Each click enhances the screen 100%!
This works quite good with lmms
Here is a screenie with 0% enhancement:
![image](https://user-images.githubusercontent.com/6662066/157958920-58bed6ab-6fa7-456b-a932-517d815509ed.png)
200%
![image](https://user-images.githubusercontent.com/6662066/157959056-87b871c8-1907-4404-b101-a7cca99c4472.png)
LMMS-UI scrolls willingly as the mouse get close to the edge, and it is significantly better for the eyes.
I believe this very simple tool is extremely valuable for sight-impaired like my self, trying to use LMMS.
